README.md

```diff
@@ -84,23 +106,22 @@ Options:
   -h, --help                        display help for command
 ```
 
-Or you can provide most of these options through a configuration file. The following files are supported:
-
-- `.rollup-type-bundlerrc`
-- `.rollup-type-bundlerrc.json`
-- `.rollup-type-bundlerrc.yaml`
-- `.rollup-type-bundlerrc.yml`
-
-The file should either be located at the current working directory from where this rollup-type-bundler is called, or provided as a custom path with `--config`.
+Or, you can set most of these options through a configuration file. This
+file should be located at your current working directory (where you're
+calling this package) or at a path you'll have to provide using
+`--config`. It should be named `.rollup-type-bundlerrc`, optionally
+suffixed with `.json`, `.yaml`, or `.yml`.
```

Comment:
```suggestion
Or, you can set most of these options through a configuration file. This
file should be located at your current working directory (where you're
calling this package). It should be named `.rollup-type-bundlerrc`, optionally
suffixed with `.json`, `.yaml`, or `.yml`.
```

My bad. but I decided to not do that option because it's too much effort lul.